e2e-tests/cypress/integration/appeal-submission-appeal-site-address.feature

```diff
@@ -0,0 +1,93 @@
+@wip
+Feature: Appellant provides the Appeal Site Address
+    I need to provide the appeal site address, so that all parties know which site the appeal is against.
+
+    Scenario Outline: Prospective Appellant submits a valid appeal site address
+        Given the user is prompted for the site address
+        When the user provides their appeal site address as <Address Line 1> and <Address Line 2> and <Town or City> and <County> and <Postcode>
+        Then the user is able to continue with the provided address
+        #And the user can see that their appeal has been updated with the provided address
+        Examples:
+            | Address Line 1  | Address Line 2 | Town or City | County       | Postcode  |
+            | "1 Taylor Road" | "Clifton"      | "Bristol"    | "South Glos" | "BS8 1TG" |
+            | "2 Taylor Road" | ""             | ""           | "South Glos" | "BS8 1TG" |
+            | "4 Taylor Road" | ""             | "Bristol"    | "South Glos" | "BS8 1TG" |
+            | "5 Taylor Road" | "Clifton"      | ""           | "South Glos" | "BS8 1TG" |
+
+    Scenario Outline: Prospective Appellant submits a site address without a mandatory field
+        Given the user is prompted for the site address
+        When the user provides their appeal site address as <Address Line 1> and <Address Line 2> and <Town or City> and <County> and <Postcode>
+        Then the user is informed that <reason>
```

Comment:
Perhaps this could read something like, "Then the user is informed that they cannot continue with the provided address <<because>>"

Reply:
Updated the example scenarios as per Frank's comment. Also updated the Postcodes are validated scenario with examples and steps